### ISSUE TITLE
Option to skip top-of page H1 link with site URL

### ISSUE THREAD

**Issue by epogrebnyak** [pages-themes/primer#40]
By default the Premier theme adds an H1 link with site URL on top of page. I believe [the code below][link] is responsible for this behaviour:

```
      {% if site.title and site.title != page.title %}
      <h1><a href="{{ "/" | absolute_url }}">{{ site.title }}</a></h1>
      {% endif %}
```

[link]: https://github.com/pages-themes/primer/blob/516c116ef77acc5ffc69345602d38b1b24da4fca/_layouts/default.html#L13-L15

I would like to find out how can top-of page link can be switched off.

My initial idea is forking a repo and deleting the code above from template. I would like to learn how can add and extra custom variable in `_config.yaml` to make H1 with URL disappear.

I think the code will look like: 

```
      {% if must_show and (if site.title and site.title != page.title %)}
      <h1><a href="{{ "/" | absolute_url }}">{{ site.title }}</a></h1>
      {% endif %}
```

My qustion is how can I introduce `must_show` or similar in `_config.yaml`

**Comment by seshrs** [comment]
I believe this was discussed in #21. 

According to [this comment](https://github.com/pages-themes/primer/issues/21#issuecomment-440742963), the initial title heading is removed if you specify the following in `_config.yml`:
```yaml
name: Page name that appears as HTML “title”
title: null
```

**Comment by epogrebnyak** [comment]
@seshrs thanks for the info  - I think I just created a duplicate. 

However - will `title: null`  shut the display of title in `<title>` tag? I cannot find where it is generated.

**Comment by seshrs** [comment]
> However - will `title: null` shut the display of title in `<title>` tag? I cannot find where it is generated.

You're right, adding `title: null` *alone* makes the `<title>` tag not render. But if you add a `name: My Page Title` in `_config.yml` in addition, the `<title>` tag renders. (I just tried this on one of my sites, and the `<title>` tag renders as expected.)

**Comment by epogrebnyak** [comment]
Do you know which part of the layout is responsible for this in the repo?
Сould not code responsible for <title> tag

вс, 16 авг. 2020 г. в 06:20, Sesh Sadasivam <notifications@github.com>:

**Comment by seshrs** [comment]
@epogrebnyak like you mentioned, the `<title>` tag is generated by the [`jekyll-seo-tag`](https://github.com/jekyll/jekyll-seo-tag/) plugin. Here's the code that uses the `name` if no `title` is provided in `_config.yml`: https://github.com/jekyll/jekyll-seo-tag/blob/933b914ab27601ca29615489f0343c4417bb146a/lib/jekyll-seo-tag/drop.rb#L33-L35

**Comment by epogrebnyak** [comment]
Maybe make a PR for README.md that has advice about avoiding the top header?
